crates/webidl/src/idl_type.rs

```diff
@@ -478,7 +549,24 @@ impl<'a> IdlType<'a> {
             },
             IdlType::Enum(name) => Some(ident_ty(rust_ident(camel_case_ident(name).as_str()))),
 
-            IdlType::Nullable(idl_type) => Some(option_ty(idl_type.to_syn_type(pos)?)),
+            IdlType::Nullable(idl_type) => {
+                match **idl_type {
+                    IdlType::Union(ref idl_types) =>
+                        if idl_types
+                            .iter()
+                            .all(|idl_type|
```

Comment:
To confirm, is this ever actually `false`? If so it seems fine to keep the branch in, but if not I think it'd be fine to change this to an `assert!` and always emit an `Object`

Reply:
Yes, unions can contain not only interfaces. See, for example, `BasePropertyIndexedKeyframe` or `IDBObjectStoreParameters`.

Reply:
Ah ok, good to know!